.github/workflows/generate_release_note.yml

```diff
@@ -0,0 +1,36 @@
+name: Auto Release Notes
```

Comment:
Can we add a `RELEASE_NOTES.md` file instead, and update it whenever a release tag is created using the existing publish actions?